### ISSUE TITLE
Upgrade to latest story-hub version

### ISSUE THREAD

**Issue by adnrs96** [storyscript/sls#66]
Some buggy things around the wrappers for services has been fixed in the hub repo but since this is still using very old hub version, the benefit hasn't been passed on.

Upgrade to use latest story-hub.

Example: this will fix the arguments suggestion not working because services wrapper code in SLS has a bug which tries to return events instead of args.

**Comment by wilzbach** [comment]
see e.g. https://github.com/storyscript/sls/pull/64

**Comment by adnrs96** [comment]
@wilzbach  I am talking about upgrading story-hub. SLS uses `asyncy-hub==0.0.1'` right now.

**Comment by wilzbach** [comment]
Yeah, it currently uses both packages (asyncy-hub and story-hub).
